### DISCUSSION TITLE
Receiver awscloudwatch

### DISCUSSION THREAD

**Discussion by WagnerCarvalho** [open-telemetry/opentelemetry-collector-contrib#20312] [Q&A]
I am testing on minikube to receive logs from CloudWatch... I am getting the following error.

awscloudwatchreceiver@v0.73.0/logs.go:215	unable to retrieve logs from cloudwatch	{"kind": "receiver", "name": "awscloudwatch", "data_type": "logs", "log group": "/aws/batch/job", "error": "NoCredentialProviders: no valid providers in chain. Deprecated.\n\tFor verbose messaging see aws.Config.CredentialsChainVerboseErrors"}

    receivers:
      awscloudwatch:
        profile: default
        region: us-east-1
        imds_endpoint: ""
        logs:
          groups:
            named:
              /aws/batch/job:
                prefixes:
                - /aws/batch/job
                names: []
          max_events_per_request: 1000
          poll_interval: 1m

image: otel/opentelemetry-collector-contrib:0.73.0

It seems to me that it's a credential error, however, when running the receiver test on my machine, the logs are being received.
Could someone help me with the error?

**Comment by lewis262626** [comment]
It's a AWS Go SDK issue. Id look at the troubleshooting guide on AWS's docs
